tests/regressiontests/test-simple.m4

```diff
@@ -17,7 +17,8 @@
 # Now we take the parsed data and assign them no nice-looking variable names,
 # sometimes after a basic validation
 if [ "$_arg_print_optionals" = on ]; then
-    echo "_supplied_arg_prefix=${_supplied_arg_prefix-x},_supplied_arg_print_optionals=${_supplied_arg_print_optionals-x},_supplied_arg_la=${_supplied_arg_la-x},_supplied_arg_not_supplied=${_supplied_arg_not_supplied-x}"
+    set -u
```

Comment:
This looks like a debug session artifact.

Reply:
This is to ensure that the arguments that were marked as supplied exist. If they do not exist then the script will fail. I think we should test this to make sure. Alternatively we can test if the `_supplied_arg_x` are set to zero above the if-statement, and exit 1 if they are not.